src/code/stores/simulation-store.coffee

```diff
@@ -101,6 +101,12 @@ SimulationStore   = Reflux.createStore
   onRunSimulation: ->
     @_runSimulation()
 
+  stepUnits: () ->
+    if @isRecordingPeriod
```

Comment:
Thanks @sfentress -- I totally missed that.  Running c07352c locally I now get new columns in CODAP when I change units.   

(we don't really want to add new columns, but that is a different story)